.pfnci/linux/tests/cuda118.multi.sh

```diff
@@ -14,5 +14,5 @@ export CUPY_ACCELERATORS="cutensor,cub"
 "$ACTIONS/build.sh"
 export OMPI_ALLOW_RUN_AS_ROOT=1
 export OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
-"$ACTIONS/unittest.sh" "not slow and multi_gpu"
+"$ACTIONS/unittest.sh" "not slow and multi_gpu" -s -v
```

Comment:
```suggestion
"$ACTIONS/unittest.sh" "not slow and multi_gpu" -s -v .
```

Reply:
The first argument is for MARKER, and the remainings are for target test files (which defaults to `.` when not specified). Passing `-s -v` here is an abuse, but it works 😅

https://github.com/cupy/cupy/blob/673485073d67d522d439a59e4b14407ed7d647c5/.pfnci/linux/tests/actions/unittest.sh#L6